pkg/component/garden/system/virtual/virtual.go

```diff
@@ -611,6 +611,34 @@ func (g *gardenSystem) computeResourcesData() (map[string][]byte, error) {
 				Name:     user.AllAuthenticated,
 			}},
 		}
+		roleReadGardenerInfoConfigMap = &rbacv1.Role{
```

Comment:
Could you please enhance and adjust the operator e2e test to check the namespace and resources?